### PR TITLE
Create Blog “how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud”

### PR DESCRIPTION
**Pull request by vinnarasu-ganesan**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by vinnarasu-ganesan** [comment]
Today we received an update from Marketing Team, that they have finalized the name for the Private cloud offering at Discover – “ HPE GreenLake for Private Cloud Enterprise “. PR is updated with the proposed name accordingly.

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+## Understanding private cloud networking
+
+The following illustration shows how you can use NSX objects to achieve NSX logical networking in HPE GreenLake for Private Cloud Enterprise.
+
+![](/img/fig-2.jpg)
+
+The network components are as follows:
+
+##### **VM**
```
Comment (choukden) [inline content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:29]: You may want to revisit the font size for these network components in this section as per Dale's suggestion. Maybe you can try heading level 4 or level 3, or a standard text in bold.

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+---
+## Introduction
+
+HPE GreenLake for Private Cloud Enterprise is: designed to deliver and help manage a private cloud. Available on the HPE GreenLake Central platform, the HPE GreenLake for Private Cloud Enterprise is
+
+* An HPE designed, implemented, owned, and operated private cloud that is deployed at a customer site
+* Offered as a consumption-based service that enables customers to better align costs to outcomes
+* An intuitive self-service portal UI used to create and manage private cloud services such as compute, storage, and network (example described in this blog)
+
+This blog post explains the steps to create a virtual network with a static IP pool and DHCP using NSX-T, network virtualization, and a security platform that enables the virtual cloud network in HPE GreenLake for private cloud.
```
Comment (choukden) [inline content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:19]: ....in HPE GreenLake for Private Cloud Enterprise.

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+![](/img/fig-8.jpg)
+
+**List View:**
+
+![](/img/fig-9.jpg)
+
+## Summary
+
+In this blog post, we covered how to get started with software-defined networking in HPE GreenLake for Private Cloud Enterprise and explained the steps to create a sample virtual network with both static IP pool and DHCP. In the next article, we will cover the NSX distributed firewall feature of HPE GreenLake for private cloud and explain how to create and enforce firewall rules to restrict the network traffic to the virtual machines.
```
Comment (choukden) [inline content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:168]: ...firewall feature of HPE GreenLake for Private Cloud Enterprise and explain...

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md (hunk 1 above)
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md]: Keeping as standard text in Bold. Hope that is fine.

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md (hunk 2 above)
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md]: Done

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md (hunk 3 above)
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md]: Done

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+**VM**
+
+Tenant virtual machines (VMs) are connected to Blue and Green networks.
+
+**NSX-T Segments**
+
+NSX-T segments are layer 2 virtual domains and there are two types of segments in an NSX-T Data Center
+
+* VLAN-backed segments: This is used for uplink traffic external to the NSX-T Data Center.
```
Comment (ma-los) [inline content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:37]: for vlan based segments, is there alignment required with the network infrastructure ? Or do we get vlan deployed automatically across the DC network infra  ?

**Review by ma-los — CHANGES_REQUESTED** [review]
Hi @vinnarasu-ganesan , some suggestions added please have an eye.

Cheers

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md (hunk 2 above)
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:19]: Reverting the name as "HPE GreenLake for private cloud" for now. I will update the name as "HPE GreenLake for Private Cloud Enterprise" after Discover announcement.

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+Gateway that processes the traffic between the logical and physical networks. A Tier-0 gateway has downlink connections to Tier1 gateways and uplink connections to the physical networks.
+
+##### **Ext-Net**
+
+Interface connected to Virtual Distributed Switch configured in a customer environment for enabling external connectivity from the tenant virtual machines.
+
+## Prerequisites
+
+Role: Customer network admin with the following role permission to enable access to NSX network objects:
```
[the remark this answers is not in the document]
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:62]: Updated the content with the steps to update the role

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+---
+## Introduction
+
+HPE GreenLake for Private Cloud Enterprise is: designed to deliver and help manage a private cloud. Available on the HPE GreenLake Central platform, the HPE GreenLake for Private Cloud Enterprise is
+
+* An HPE designed, implemented, owned, and operated private cloud that is deployed at a customer site
+* Offered as a consumption-based service that enables customers to better align costs to outcomes
+* An intuitive self-service portal UI used to create and manage private cloud services such as compute, storage, and network (example described in this blog)
+
+This blog post explains the steps to create a virtual network with a static IP pool and DHCP using NSX-T, network virtualization, and a security platform that enables the virtual cloud network in HPE GreenLake for Private Cloud Enterprise.
```
[the remark this answers is not in the document]
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md]: Modified as "This blog post explains how a Customer Network Administrator can create a virtual network with a static IP pool and DHCP using NSX-T, network virtualization, and a security platform that enables the virtual cloud network in HPE GreenLake for private cloud."

File: content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md (hunk 4 above)
Comment (vinnarasu-ganesan) [reply content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md:37]: Added a statement: Note: Raise HPE Support case to enable the backend infrastructure to support this type.

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Vinnarasu, 

Is the blog post now ready for publication? r do you still need to collect feedback from other members of your team, or do you still have some suggestions for edit to incorporate?

Thanks
Denis

**Comment by vinnarasu-ganesan** [comment]
@choukden I incorporated all the requested changes. If reviewers are okay, the blog can be published.

**Comment by vinnarasu-ganesan** [comment]
@reddypramod85 I see you have closed this PR without merging it. Is there any particular reason for closing this PR without publishing the blog?

**Comment by vinnarasu-ganesan** [comment]
Incorporated changes requested by @drensing

**Pull request merged**